rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,18 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mocked_spec_object(monkeypatch, spec_attributes):
+    spec_attributes['save'] = lambda: None
+    spec = SpecFile.__new__(SpecFile)
+    for attribute, value in spec_attributes.items():
+        monkeypatch.setattr(spec, attribute, value, raising=False)
```

Comment:
What's the point of setting the `spec_content` attribute twice? Wouldn't it be better to convert `value` first if `attribute` is `'spec_content'`, and only then call `monkeypatch.setattr()`?